addons/sys_core/stringtable.xml

```diff
@@ -126,5 +126,25 @@
             <Japanese>ヘッドセットのトグル/スペクテイターのトグル</Japanese>
             <Italian>Attiva/Disattiva Cuffia - Attiva/Disattiva Spettatori</Italian>
         </Key>
+        <Key ID="STR_ACRE_sys_core_difficultyInterference_displayName">
+            <English>Model Interference?</English>
+            <Spanish>Modelar interferencias?</Spanish>
+        </Key>
+        <Key ID="STR_ACRE_sys_core_difficultyFullDuplex_displayName">
+            <English>Full Duplex radios?</English>
+            <Spanish>Radios Dúplex?</Spanish>
+        </Key>
+        <Key ID="STR_ACRE_sys_core_difficultyAntennaDirection_displayName">
+            <English>Ignore Antena Direction?</English>
+            <Spanish>Ignorar dirección de antena?</Spanish>
+        </Key>
+        <Key ID="STR_ACRE_sys_core_difficultyTerrainLoss_displayName">
+            <English>Terrain Loss Coefficient</English>
+            <Spanish>Coefficiente de pérdida de señal por terreno</Spanish>
+        </Key>
+        <Key ID="STR_ACRE_sys_core_difficultyRevealToAI_displayName">
+            <English>Reveal to AI?</English>
```

Comment:
Suggestion: `AI Voice Detection` (or equivalent)

There's already context that this setting is related to ACRE due to the `"ACRE2"` category and there's no need to phrase the title as a question (thought maybe that's just my personal preference) since it's self evident that it's a setting - if more context was needed then I'd suggest using the optional inclusion of a tooltip.